packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -336,13 +365,27 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 			this.isMultiSortEnabled = StarkComponentUtil.isInputEnabled(this.multiSort);
 		}
 
-		if (changes["multiSelect"]) {
-			this.isMultiSelectEnabled = StarkComponentUtil.isInputEnabled(this.multiSelect);
-
-			if (this.isMultiSelectEnabled && this.displayedColumns) {
-				this.displayedColumns.unshift("select");
+		if (changes["rowsSelectable"]) {
+			if (this.rowsSelectable) {
+				if (!this.displayedColumns.includes("select")) {
+					this.displayedColumns.unshift("select");
+				}
+			} else {
+				const i: number = this.displayedColumns.indexOf("select");
+				this.displayedColumns.splice(i);
 			}
 		}
+
+		if (changes["multiSelect"]) {
+			this.selection = new SelectionModel<object>(this.isMultiSelectEnabled, []);
```

Comment:
I think we might have a problem here:  we re-assign the `SelectionModel` here meaning that the subscription we did on the `ngOnInit` will still be on the old `SelectionModel` right?